server/tests/2-recipes.js

```diff
@@ -185,7 +185,7 @@ describe('/GET Search recipe by ingredient', () => {
       .set('Accept', 'application/json')
       .set('x-access-token', token)
       .end((err, res) => {
-        expect(res.statusCode).to.equal(201);
+        expect(res.statusCode).to.equal(200);
         expect(res.body.success).to.equal(true);
         expect(res.body.recipes.length).to.equal(3);
```

Comment:
What you did here is not enough. Again test for each of the three recipes and make sure each contains appropriate data